src/fastener.js

```diff
@@ -159,6 +167,9 @@ export function last(z) {const u = up(z); return u && downLast(u)}
 
 export const toZipper = focus => ({focus})
 
+const zipperPathU = (ks, focus) => downPathU(ks, toZipper(focus)) || toZipper()
```

Comment:
My thinking w/ `|| toZipper()` was to return an empty zipper (just a hole I guess). See my comment in the PR description.
I agree with you about removing `zipperPath`. I originally thought of it as an analog to Ramda's `lensPath` but the analogy is broken. I wasn't totally comfortable with it when I pushed it.